cmd/load-runner/helm.go

```diff
@@ -0,0 +1,56 @@
+package main
+
+import (
+	"context"
+	"fmt"
+)
+
+type HelmClient interface {
+	Install(ctx context.Context, config *LoadTestConfig) error
+	Upgrade(ctx context.Context, config *LoadTestConfig, phase RunPhase) error
+	Uninstall(config *LoadTestConfig) error
+}
+
+type DefaultHelmClient struct {
```

Comment:
Why do you export this, declare an interface and return the struct from the constructor?
I mean if you didn't have an interface then I'd understand but if you define an interface then I'd expect the type to be unexported and have the constructor return the interface. Thoughts?